nodes.py

```diff
@@ -879,13 +879,16 @@ class CLIPVisionLoader:
     @classmethod
     def INPUT_TYPES(s):
         return {"required": { "clip_name": (folder_paths.get_filename_list("clip_vision"), ),
+                             "device": (["GPU", "CPU"], ),
```

Comment:
to prevent breakages with existing workflows, this would need to be added under `optional`